asdf_astropy/tests/__init__.py

```diff
@@ -0,0 +1,7 @@
+import astropy
```

Comment:
Would you put this into a submodule? How about `asdf_astropy.tests.versions` based on the previous suggestions.

Reply:
Done.